src/js/disability-benefits/routes.jsx

```diff
@@ -12,6 +12,10 @@ import AdditionalEvidencePage from './containers/AdditionalEvidencePage.jsx';
 import ClaimEstimationPage from './containers/ClaimEstimationPage.jsx';
 
 const routes = [
+  <Redirect
```

Comment:
Nope, that all makes sense, I just wasn't sure that the redirect would actually work given that we use a base url. But it looks like it does, though I think we should change the `from` to:

`from="/disability-benefits/track-claims*"`

So that we catch any of the old routes, since those won't work anymore.